src/components/blog/blog-grid.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import BlogCard from './blog-card'
+import { ChevronDown } from 'react-feather'
+
+const BlogGrid = () => {
+  return (
+    <div className='container' id='blog-grid'>
+      <div className='row'>
+        <BlogCard />
```

Comment:
Are we planning to change this to dynamically pull the blogs in the future? Maybe we should add a todo here to indicate that

Reply:
Yup, they should be set to be pulled dynamically on build based on the MD files right now.